### ISSUE TITLE
Xiaomi Multimode Gateway 2 cannot be controlled in Mi Home App after being integrated in with Hass

### ISSUE THREAD

**Issue by rosystain** [AlexxIT/XiaomiGateway3#1151]
Hi.

I recently purchased a Xiaomi Multimode Gateway 2, but I encountered some things I couldn't understand.

> Q. Will the Zigbee devices continue to work in Mi Home?
A. Yes. If you do not enable ZHA or z2m mode, the devices will continue to work in Mi Home. And you can use automation in both Mi Home and Hass.

According to the FAQ, I should be able to control my gateway and its sub-devices simultaneously in both HASS and Mi Home.

But once I enable Xiaomi Gateway 3 in HASS, I can no longer control my gateway hub in Mi Home.

I'm unable to add any new devices in the Mi Home app. The indicator lights don't respond to my actions in the app, and pairing is unsuccessful.

I am also unable to control any BLE devices connected to the gateway through the Mi Home app. I have a BLE mesh wall switch, and any commands in the Mi Home app timeout and fail, but they work fine in XiaomiGateway3.

Similarly, all Zigbee devices show as offline in Mi Home after a period of time, but they continue to function correctly in HASS.

I am also using a Xiaomi MIOT integration to control Wi-Fi devices. It can also integrate BLE devices, so I quickly discovered that the behavior of the integrated BLE devices is the same as in Mi Home. When Xiaomi Gateway 3 takes over the gateway hub, both of them become unusable.

I am not using ZHA or Z2M. But I did install and try them previously. I'm not sure if this is a useful clue.

The firmware version of the gateway I'm using is 1.0.6_0016.  The current HASS system version I am using is Supervisor 2023.09.2.  And I installed Xiaomi Gateway 3 from HACS.

I would be glad to know if there's any additional information needed.

**Comment by rosystain** [comment]
#1017 

It looks very much like what this issue describes.

**Comment by AlexxIT** [comment]
You should reboot, or better reset gateway after ZHA/z2m

**Comment by rosystain** [comment]
I have already reset the gateway, and rebooting hasn't helped at all.

I only manage to see the devices reconnect in the Mijia app after restarting the gateway when Xiaomi Gateway 3 is disabled.

And once I re-enable Xiaomi Gateway 3, I can clearly notice that the Bluetooth section of the gateway in the app is no longer available.

It takes a long time to load, and then it shows that the Bluetooth gateway is not enabled.

<details><summary>

![Clipboard 2023年10月16日 12 22](https://github.com/AlexxIT/XiaomiGateway3/assets/27495128/2d32f246-489a-472a-8de1-341d36bf13b0)

![Clipboard 2023年10月16日 12 22(1)](https://github.com/AlexxIT/XiaomiGateway3/assets/27495128/9d0b60c5-ac5e-434a-a685-fa4bb82105e1)

**Comment by hbsgithub** [comment]
Same question.

Have you found the solution?

> I have already reset the gateway, and rebooting hasn't helped at all.
> 
> I only manage to see the devices reconnect in the Mijia app after restarting the gateway when Xiaomi Gateway 3 is disabled.
> 
> And once I re-enable Xiaomi Gateway 3, I can clearly notice that the Bluetooth section of the gateway in the app is no longer available.
> 
> It takes a long time to load, and then it shows that the Bluetooth gateway is not enabled.
> 
> Pic
> This usually doesn't affect the devices working properly in Home Assistant, except for the Aqara Smart Lock D100 BLE.
> 
> Just as the D100 doesn't appear in the images above. It only appears in this list after I disable Xiaomi Gateway 3 and restart the gateway. After I enable Xiaomi Gateway 3, it can still connect successfully in Home Assistant for a few days, maybe two or three days, maybe just one day.
> 
> This is the most critical issue I've encountered. I actually don't care about the Mijia app at all, but it seems that the BLE communication of D100 is heavily dependent on Mijia.

**Comment by rosystain** [comment]
> Same question.
> 
> Have you found the solution?
> 
> > I have already reset the gateway, and rebooting hasn't helped at all.
> > I only manage to see the devices reconnect in the Mijia app after restarting the gateway when Xiaomi Gateway 3 is disabled.
> > And once I re-enable Xiaomi Gateway 3, I can clearly notice that the Bluetooth section of the gateway in the app is no longer available.
> > It takes a long time to load, and then it shows that the Bluetooth gateway is not enabled.
> > Pic
> > This usually doesn't affect the devices working properly in Home Assistant, except for the Aqara Smart Lock D100 BLE.
> > Just as the D100 doesn't appear in the images above. It only appears in this list after I disable Xiaomi Gateway 3 and restart the gateway. After I enable Xiaomi Gateway 3, it can still connect successfully in Home Assistant for a few days, maybe two or three days, maybe just one day.
> > This is the most critical issue I've encountered. I actually don't care about the Mijia app at all, but it seems that the BLE communication of D100 is heavily dependent on Mijia.

No, it wasn't resolved, but I purchased another ZNDMWG03LM (the one without an Ethernet port) later. I'm not sure if this is a problem specific to the second generation (DMWG03LM) , but the issue did not recur on the later device.
